packages/basset/src/mock_querier.rs

```diff
@@ -128,6 +128,10 @@ impl WasmMockQuerier {
                             api.addr_canonicalize(&String::from("stluna_token"))
                                 .unwrap(),
                         ),
+                        airdrop_withdrawal_account: Some(
+                            api.addr_canonicalize(&String::from("airdrop_withdrawal_account"))
```

Comment:
Again, `&String::from("")` seems to be redundant, we could just put a string literal here.